package.json

```
@@ -51,11 +62,11 @@
     "webpack": "^4.35.3",
     "webpack-cli": "^3.3.6",
     "webpack-dev-middleware": "^3.7.0",
-    "webpack-dev-server": "^3.7.2",
+    "webpack-dev-server": "^3.10.1",
     "webpack-hot-middleware": "^2.25.0"
   },
   "bin": {
-    "draft": "./bin/launch.js"
+    "draft": "./launch.prod.js"
```

Why did you move the path? Is it not still in `bin`? 
```suggestion
    "draft": "./bin/launch.prod.js"
```
I think this is what broke the build.

---

Should fix: 
> - Running "npm run draft" doesn't work for some reason - you have to use node to run the launch file

---

Good catch.

---

The error received is that it can't find the command "draft", rather than it not being able to find the file. So this doesn't fix the error, but it does need to be fixed.